cypress.config.ts

```diff
@@ -0,0 +1,36 @@
+import { defineConfig } from 'cypress';
+
+export default defineConfig({
+	viewportWidth: 1500,
+	viewportHeight: 860,
+	video: false,
+	failOnStatusCode: false,
+	chromeWebSecurity: false,
+	experimentalSessionSupport: true,
+	blockHosts: [
+		'*ophan.theguardian.com',
+		'pixel.adsafeprotected.com',
+		'*permutive.com',
+		'*adnxs.com',
+		'*adsystem.com',
+		'*casalemedia.com',
+		'*pubmatic.com',
+		'*360yield.com',
+		'*omnitagjs.com',
+		'*the-ozone-project.com',
+		'*openx.net',
+	],
+	fixturesFolder: false,
+	retries: {
+		runMode: 2,
+		openMode: 0,
+	},
+	e2e: {
+		// We've imported your old cypress plugins here.
+		// You may want to clean this up later by importing these.
+		setupNodeEvents(on, config) {
+			return require('./cypress/plugins/index.ts')(on, config);
+		},
+		baseUrl: 'http://localhost:9234/',
```

Comment:
looks like this is a new port to run on?

Reply:
![image](https://github.com/guardian/manage-frontend/assets/114918544/c0baa5d8-c59e-4c3e-a8e1-3b10b58182ec)

looks same as before?

Reply:
hmm my storybook goes to `localhost:6006` Is it a different port?